samples/pmic/native/npm1300_fuel_gauge/README.rst

```diff
@@ -72,6 +77,7 @@ To connect your DK to the nPM1300 EK, complete the following steps:
    * On the **P17** pin header, connect all LEDs with jumpers.
    * On the **P13** pin header, connect **RSET1** and **VSET1** pins with a jumper.
    * On the **P14** pin header, connect **RSET2** and **VSET2** pins with a jumper.
+   * On the **P18** pin header, disconnect **VDDIO** jumper.
```

Comment:
I think it's better to add "Remove all existing connections" at the start of this list, the same way as we have in One Button sample: https://github.com/nrfconnect/sdk-nrf/blob/main/samples/pmic/native/npm1300_one_button/README.rst?plain=1#L38